cpp/src/jit/cache.cpp

```diff
@@ -150,8 +157,13 @@ std::string cudfJitCache::cacheFile::read()
     return std::string();
   }
 
+  // Create args for file locking
+  flock fl{};
+  fl.l_type   = F_RDLCK;  // Shared lock for reading
+  fl.l_whence = SEEK_SET;
+
   // Lock the file descriptor. we the only ones now
```

Comment:
This comment does not seem correct now, with the lock changes.

Reply:
You’re right. Thanks for the quick review